src/js/edu-benefits/1990e/config/form.js

```diff
@@ -6,8 +6,9 @@ import {
 import fullSchema1990e from 'vets-json-schema/dist/transfer-benefits-schema.json';
 
 import * as currentOrPastDate from '../../../common/schemaform/definitions/currentOrPastDate';
-import * as fullName from '../../../common/schemaform/definitions/fullName';
-import * as ssn from '../../../common/schemaform/definitions/ssn';
+import * as fullNameCommon from '../../../common/schemaform/definitions/fullName';
```

Comment:
Yeah, I think that makes sense for cases where the front end does not need to modify the schema. But I don't think we should modify all the common components this PR uses now, though. I think that's better as a separate issue.

The fullName schema is a little tricky because we might need to modify it depending on if the name is required or not.

For now I think we could import it like this to make it easy to change to the above style in the future:

`import { uiSchema as fullNameUiSchema } from '../../../common/schemaform/definitions/fullName';`